hack/test/capi/cluster-aws.yaml

```diff
@@ -57,7 +57,7 @@ kind: TalosControlPlane
 metadata:
   name: talos-e2e-{{TAG}}-aws-controlplane
 spec:
-  version: v1.19.4
+  version: v1.19.3
```

Comment:
is that intended?

Reply:
kind of :) it is broken now for 0.7, fixed with another PR https://github.com/talos-systems/talos/pull/2784

so this fixes e2e for 0.7, if we backport #2784, we can fix e2e for 0.7 with 1.19.4